common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -225,6 +225,51 @@ def test_notification_updates(self):
         self.assertFalse(problem_page.is_save_notification_visible())
 
 
+class ProblemSaveStatusUpdateTests(ProblemsTest):
+    """
+    Tests the problem status updates correctly with an answer change and save.
+    """
+    def get_problem(self):
+        """
+        Problem structure.
+        """
+        xml = dedent("""
+            <problem>
+                <label>Which of the following countries has the largest population?</label>
+                    <multiplechoiceresponse>
+                      <choicegroup type="MultipleChoice">
+                        <choice correct="false">Brazil <choicehint>timely feedback -- explain why an almost correct answer is wrong</choicehint></choice>
+                        <choice correct="false">Germany</choice>
+                        <choice correct="true">Indonesia</choice>
+                        <choice correct="false">Russia</choice>
+                      </choicegroup>
+                    </multiplechoiceresponse>
+            </problem>
+        """)
+        return XBlockFixtureDesc('problem', 'TEST PROBLEM', data=xml,
+                                 metadata={'max_attempts': 10},
+                                 grader_type='Final Exam')
+
+    def test_status_removed_after_save_before_submit(self):
+        """
+        Verifies that the status on a problem is removed after a save and before a second submission
+        """
```

Comment:
@staubina Please add BDD style docstring. You can see an example [here](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/video/test_video_module.py#L468).